src/components/interfaces/MOBILE_API.xml

```diff
@@ -1853,14 +1853,26 @@
         <param name="compassDirection" type="CompassDirection" mandatory="true">
             <description>See CompassDirection.</description>
         </param>
-        <param name="pdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true">
-            <description>PDOP.  If undefined or unavailable, then value shall be set to 0.</description>
-        </param>
-        <param name="hdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true">
-            <description>HDOP.  If value is unknown, value shall be set to 0.</description>
-        </param>
-        <param name="vdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true">
-            <description>VDOP.  If value is unknown, value shall be set to 0.</description>
+        <param name="pdop" type="Float" minvalue="0" maxvalue="1000" mandatory="false" since="5.0">
+            <description>PDOP.</description>
+            <history>
+                <param name="pdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true" until="5.0">
+                </param>
+            </history>
+        </param>
+        <param name="hdop" type="Float" minvalue="0" maxvalue="1000" mandatory="false" since="5.0">
+            <description>HDOP.</description>
+            <history>
+                <param name="hdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true" until="5.0">
+                </param>
+            </history>
+        </param>
+        <param name="vdop" type="Float" minvalue="0" maxvalue="1000" mandatory="false" since="5.0">
+            <description>VDOP.</description>
+            <history>
+                <param name="vdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true" until="5.0">
+                </param>
+            </history>
         </param>
         <param name="actual" type="Boolean" mandatory="true">
```

Comment:
please update the Mobile API to make sure all other fields under `GPSData` are not mandatory. See https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md for reference

Reply:
@yang1070 Thank you. Fixed in 184ef635260373b1cb7b193010c77e18f7c0f8cc

Reply:
@yang1070 Do we need to make the following changes ??
```
<struct name="VehicleDataResult">
    <description>Individual published data request result</description>
    <param name="dataType" type="VehicleDataType">
      <description>Defined published data element type.</description>
    </param>
    <param name="resultCode" type="VehicleDataResultCode">
      <description>Published data result code.</description>
    </param>
  </struct>
```
to 
```
<struct name="VehicleDataResult">
    <description>Individual published data request result</description>
    <param name="dataType" type="VehicleDataType" since="5.0">
        <description>Defined published data element type.</description>
        <history>
            <param name="dataType" type="VehicleDataType" mandatory="true" until="5.0">
            </param>
        </history>
    </param>
    <param name="resultCode" type="VehicleDataResultCode" since="5.0">
        <description>Published data result code.</description>
        <history>
            <param name="resultCode" type="VehicleDataResultCode" mandatory="true" until="5.0">
            </param>
        </history>
    </param>
</struct>
```
as it is written in **Proposed Mobile API changes:** of  https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md